src/app/pages/PhotoGalleryPage/__snapshots__/index.test.jsx.snap

```diff
@@ -2897,71 +2897,6 @@ exports[`Photo Gallery Page snapshots should match snapshot for PGL with all CPS
   height: 0.8125rem;
 }
 
-.c21 {
```

Comment:
Curious about why there are changes to this snapshot 🤔. As far as I can tell no changes have been made to `/pages/PhotoGalleryPage`

Reply:
They are further down https://github.com/bbc/simorgh/pull/6663/files#diff-db7ebd6ecd0b8e5ac1739cd0f9e34cf9R3034, just the generated class name has changed.